apps/projects/contracts/Projects.sol

```diff
@@ -673,7 +677,7 @@ contract Projects is AragonApp, DepositableStorage {
     }
 
     /**
-     * @notice Fund issues open to submissions from anyone: `_description`
+     * @notice Fund open submission issues: `_description`
```

Comment:
Why this change?

Reply:
We are calling it "open submission" in projects UI, and it shortens the text.

Reply:
Would a hyphen make sense? "Fund open-submission issues"?